app/docker/views/containers/edit/container.html

```diff
@@ -120,7 +120,8 @@
                   message="'Webhook (or callback URI) used to automate the recreate this container. Sending a POST request to this callback URI (without requiring any authentication) will pull the most up-to-date version of the associated image and recreate this container.'"
                 ></portainer-tooltip>
                 <label class="switch box-selector-item limited business" style="margin-left: 20px">
-                  <input disable-authorization="DockerContainerUpdate" type="checkbox" ng-model="WebhookExists" disabled="disabled" ng-checked="true" /><i></i>
+                  <input disable-authorization="DockerContainerUpdate" type="checkbox" ng-model="WebhookExists" disabled="disabled" ng-checked="true" />
+                  <span class="slider round"></span>
```

Comment:
what is this for?

Reply:
I think this is the toggle switch for webhook

Reply:
yeah